### DISCUSSION TITLE
Not able to sign up for student developer pack

### DISCUSSION THREAD

**Discussion by pstrings** [community/community#72673] [GitHub Education]
### Hi everyone,

I enrolled in an online course due to lockdown, now I'm not even in the same country as my school. But during the application it's taking location access and saying that I'm not on the same location. What can I do about this? I already provided my school issued email as well.

**Comment by Gabriel-Damas** [comment]
If you are having trouble accessing an online course due to location restrictions, you can try the following approaches: 1. **Contact technical support:** Contact the online learning platform's technical support and explain your situation. They can offer specific guidance or solutions for dealing with localization issues. Online educational institutions often have policies for dealing with regions with students who have moved to different regions or countries. 2. **Use a virtual private network (VPN):** A VPN can help mask
